app/util/package_version.py

```diff
@@ -1 +1 @@
-version = '0.0.0'  # This should not be set to a real version. This file is modified during freeze.
+version = "0.5.197-b3b126e-mod"  # DO NOT COMMIT
```

Comment:
WHAT HAVE YOU DONE?!

Just kidding. I should probably add a lint rule for the phrase "DO NOT COMMIT".